### PR TITLE
Configuration for travis ci build

### PR DESCRIPTION
**Pull request by bendiggle**
- Added dotfiles
- Added travis build file
- Added READMEs to each view directory because bootstrap needs in for this build.
- Remove dotfiles dependancy 
- Added acceptance tests run script
- updated codecept
- added phantomjs and webdriver dependancy

### CODE REVIEW

**Comment by easternbloc** [comment]
@bendiggle we should start to test everything in a container where possible as that's how our code runs in the wild.

Unit tests etc are fine to run bare bones on travis but acceptance etc should really be running in a built container.

See GRO for more details how we're achieving this in travis https://github.com/UKHomeOffice/gro/blob/master/.travis.yml. Moving along we won't be using travis for this I don't think as we can achieve a nicer containerised pipeline using drone.

**Comment by bendiggle** [comment]
@gxxm @easternbloc @joefitter This is ready now

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+services:
+  - docker
+  - redis
+
+before_install:
+  - if [[ `npm -v` != 3* ]]; then npm i -g npm@3; fi
+  - npm cache clean
+  - docker build --tag travis .
+  - docker run -d --net=host -e NODE_ENV=docker travis
```
Comment (easternbloc) [inline .travis.yml:15]: 👌

**Comment by easternbloc** [comment]
`apps/application-enquiry/views/README.md` no changes... what's going on ere then?

**Comment by joefitter** [comment]
apart from @easternbloc 's valid comment LGTM

**Comment by joefitter** [comment]
(it needs removing)

**Comment by joefitter** [comment]
ah - or have you not added any files yet - ignore if so

**Comment by bendiggle** [comment]
@joefitter @easternbloc I haven't added any files to the the views directories yet. And annoyingly they need a file in order for bootstrap to happy!

**Comment by joefitter** [comment]
yeah sorry I thought you had added files. It needs removing when we do. merging!

**Pull request merged**